index.bs

```diff
@@ -4115,6 +4147,810 @@ opened</dfn> steps given |window|, |type| and |message|.
+
+1. Let |status| be |response|'s [=response/status=].
+
+1. Let |status text| be |response|'s [=status message=].
+
+1. If |response|'s [=cache state=] is "<code>local</code>", let |from cache| be
+   true, otherwise let it be false.
+
+1. Let |headers| be an empty list.
+
+1. Let |mime type| be the [=essence=] of the [=computed mime type=] for |response|.
+
+   Note: this is whatever MIME type the browser is actually using, even if it
+   isn't following the exact algorithm in the [[MIMESNIFF]] specification.
+
+1. For each (|name|, |value|) in |response|'s [=response/headers list=]:
+
+  1. Append the result of [=get a header=] with |name| and |value| to |headers|.
+
+1. Let |bytes received| be the total number of bytes transmitted as part of the
+   HTTP response associated with |response|.
+
+1. Let |headers size| be the number of bytes transmitted as part of the header
+   fields section of the HTTP response.
+
+1. Let |body size| be |response|'s [=response body info=]'s [=encoded size=].
+
+1. Let |content| be |response|'s [=get the response content info=] with |response|.
+
+1. Return a [=/map=] matching the <code>network.ResponseData</code> production,
+   with the <code>url</code> field set to |url|, the <code>protocol</code> field
+   set to |protocol|, the <code>status</code> field set to |status|, the
+   <code>statusText</code> field set to |status text|, the
+   <code>fromCache</code> field set to |from cache|, the <code>headers</code>
+   field set to |headers|, the <code>mimeType</code> field set to |mime type|, the
+   <code>bytesReceived</code> field set to |bytes received|, the
+   <code>headersSize</code> field set to |headers size|, the
+   <code>bodySize</code> field set to |body size|, and the <code>content</code>
+   field set to |content|.
+
+</div>
+
+### Events ### {#module-network-event}
+
+#### The network.beforeRequestSent Event #### {#event-network-beforeSendRequest}
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        network.BeforeRequestSent = {
+         method: "network.beforeRequestSent",
+         params: network.BeforeRequestSentParameters
+       }
+
+       network.BeforeRequestSentParameters = {
+         network.BaseParameters,
+         initiator: network.Initiator,
+       }
+      </pre>
+   </dd>
+</dl>
+
+This event is emitted before a request is sent (either over the network or
+before it's handled by a serviceworker or a local cache).
+
+<div algorithm>
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi before
+request sent</dfn> steps given |request|:
+
+1. If [=before request sent map=] does not contain |request|, set [=before
+   request sent map=][|request|] to a new set.
+
+1. Let |redirect count| be |request|'s [=redirect count=].
+
+1. Add |redirect count| to [=before request sent map=][|request|].
+
+1. Let |is redirect| be true if |request|'s [=redirect count=] is greater than
+   0 or false otherwise.
```

Comment:
Does it make sense to have `isRedirect`, if it's always `redirectCount>0`?

Reply:
I don't mind. @juliandescottes wdyt?

Reply:
Sounds good, let's keep only `redirectCount`

Reply:
I assume we can now close this issue given that the changes have already been applied.